tools/pipelines/test-real-service.yml

```diff
@@ -45,6 +45,7 @@ stages:
         testWorkspace: ${{ variables.testWorkspace }}
         artifactBuildId: $(resources.pipeline.client.runID)
         testCommand: test:realsvc:local:report:full
+        timeoutInMinutes: 75
```

Comment:
I think we actually want this between lines 63 and 64 instead of here.

Reply:
Right right! Thanks for catching this up.